src/components/AlertCard/styles.module.css

```diff
@@ -6,20 +6,24 @@
 @value gray from layout;
 
 .card {
-  /* padding: 0.5rem; */
-  background-color: #f0f0f0;
+  background-color: white !important;
```

Comment:
This value is set in a number of overlapping places - here, in `.card-header`, and in `.columns`. I think it'd be best to:
1. Change this value to `transparent` instead, since the element behind it is white
2. Find one class that if it gets this value, makes the desired impact (I think it's `.card-header`), and just have it be there.

Reply:
@kem247 just leaving a comment here again, so you can see this. Doesn't look like it changed yet.

Reply:
fixed